pom.xml

```diff
@@ -156,10 +156,20 @@
     </pluginManagement>
 
     <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
+          <target>11</target>
+        </configuration>
+      </plugin>
```

Comment:
Is this change actually required? I believe the default configuration (inherited from the vertx parent) is JDK11 (in this branch of course)